onward/app/controllers/MostViewedGalleryController.scala

```diff
@@ -27,7 +28,6 @@ class MostViewedGalleryController(
   private val dataId: String = "multimedia/gallery"
   private val config = CollectionConfig.empty.copy(
     displayName = Some("More galleries"),
-    groups = Some(List("multimedia/gallery")),
```

Comment:
Not exactly sure where this is used, but should this be saying groupsConfig, instead? Or is it ok to just delete?